EFCountingLabel/Classes/EFCount.swift

```diff
@@ -47,7 +47,42 @@ extension EFCount {
 }
 
 public class EFCounter {
+
+    public enum RefreshRate: Int {
+        case _5fps = 5
+        case _6fps = 6
+        case _10fps = 10
+        case _12fps = 12
+        case _15fps = 15
+        case _20fps = 20
+        case _30fps = 30
+        case _60fps = 60
+        case max = 0
+
+        private var interval: Int {
+            switch self {
```

Comment:
If you keep it as a enum, then it's better to use the rawValue:
```swift
let rawValue = self.rawValue
return rawValue == 0 ? 1 : Swift.max(1, 60 / rawValue)
```

If you change `refreshRate` to an Int, then it's just:
```swift
fileprivate extension Int {
    var interval: Int {
        return self == 0 ? 1 : max(1, 60 / self)
    }
}
```
(or a static func, that works as well)